css/modules.css

```diff
@@ -0,0 +1,22 @@
+.heading-main {
+  /*float: left;*/
+  padding: 0 0 .2em 1em;
+}
+
+.icon-menu {
+  margin-right: 1em;
+  width: 1.5em;
+}
+
+@media only screen and (min-width:640px) {
+  .icon-menu {
+    display: none;
+  }
+  .main-nav ul {
+    display: flex;
+    flex-direction: row;
+  }
+  .main-nav:hover ul {
+    display: none;
+  }
+}
```

Comment:
This is making your nav menu disappear and flicker. The nav menu is non-functional above 640px.